<launch>

    <!--    <machine name="local_alt" address="localhost" default="true"/>-->
    <param name="Voc_file" type="string" value="$(find ORB_SLAM3)/../../../Vocabulary/ORBvoc.txt"/>
    <param name="config_file" type="string" value="$(find ORB_SLAM3)/config/Urban.yaml"/>
<!--    <param name="do_rectify" type="bool" value="false"/>-->
    <param name="img_l" type="string" value="/stereo/left/image_mono"/>
    <param name="img_r" type="string" value="/stereo/right/image_mono"/>


    <param name="dataset" type="string" value="urban"/>
    <!--    traj_format 0:kitti 1:tum-->
    <param name="traj_format" type="int" value="1"/>
    <param name="traj_outpath" type="string" value="$(find ORB_SLAM3)/results/"/>


    <node name="Stereo_Inertial_pub" pkg="ORB_SLAM3" type="Stereo_Inertial_pub" output="screen" cwd="node" required="true"/>

<!--    &lt;!&ndash; 启动rviz &ndash;&gt;-->
<!--    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ORB_SLAM3)/config/rviz.rviz" />-->

<!--    <node pkg="rosbag" type="record" name="record"-->
<!--          args="/orb/path /orb/odom /camera/image_raw/compressed -O orb_stereo_imu_urban.bag"/>-->
</launch>